Gemfile.lock

```diff
@@ -398,7 +398,7 @@ GEM
     ruby-ole (1.2.11.6)
     ruby-prof (0.14.2)
     ruby-progressbar (1.4.2)
-    rubyzip (1.1.6)
+    rubyzip (1.1.7)
```

Comment:
Ok, but selenium webdriver is not updated, I guess? But the ffi is, which is probably something related.